backend/cassandra/session.go

```diff
@@ -137,6 +137,34 @@ func (s *Session) GetTables(keyspace string) ([]string, error) {
 	return tables, nil
 }
 
+// GetTables queries the cassandra cluster for a list of an existing tables in a given keyspace.
+func (s *Session) GetVariables(ctx context.Context, query string) ([][]string, error) {
+	iter := s.session.Query(query).WithContext(ctx).Iter()
+
+    var variables [][]string
+
+	// check if one or two columns (are labels present?)
+	if (len(iter.Columns()) == 1) {
+		var column string
+		for iter.Scan(&column) {
+			variables = append(variables, []string{"", column})
+		}
+	} else if (len(iter.Columns()) == 2) {
+		var column1, column2 string
```

Comment:
Having variables named for their purpose would be more convenient, like `var label, value string`. Again, in `plugin` context it would be more natural because the `plugin` knows about high-level purpose of received data, unlike the `cassandra` does.